lostphotosfound/server.py

```diff
@@ -262,8 +262,7 @@ def lostphotosfound(self):
                 header_subject = _charset_decoder(mail['Subject'])
                 print '[%s]: %s' % (header_from, header_subject)
 
-		# use raw header, header_from sometimes excludes the email address
-		sender = email.utils.parseaddr(mail['From'])[1]
+		sender = email.utils.parseaddr(header_from)[1]
```

Comment:
Any particular reason for this change given the previous comment?
"# use raw header, header_from sometimes excludes the email address"